ui/apps/platform/src/types/group.proto.ts

```diff
@@ -21,3 +21,13 @@ export type GroupProperties = {
     key: string;
     value: string;
 };
+
```

Comment:
Commenting on the empty line, does `Group` type above need `traits: Traits` property?

If yes, you know better than me:
* Is it optional in response? Then `traits?:`
* Is null a possible value? Then `Traits | null`